src/manifolds/PowerManifold.jl

```diff
@@ -82,13 +82,17 @@ function PowerManifold(
 end
 
 @doc raw"""
-    PowerMetric <: Metric
+    PowerMetric(metric::metric)
 
 Represent the [`Metric`](@ref) on an [`AbstractPowerManifold`](@ref), i.e. the inner
 product on the tangent space is the sum of the inner product of each elements
 tangent space of the power manifold.
+
+It is constructed based on the given `metric` on the powered space.
```

Comment:
Could you the explicit constructor signature here?

Reply:
OK, done.